### ISSUE TITLE
Rerun role creates extra license_key property

### ISSUE THREAD

**Issue by rkoncier** [newrelic/infrastructure-agent-ansible#68]
Rerun of role creates additional line with 'license_key' property. Property is empty then start of newrelic agent service fails .

Here is cat of /etc/newrelic-infra.yml file

`license_key: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
log_file: /var/log/newrelic-infra/nrsysmond.log
log_to_stdout: false

license_key:`

![image](https://user-images.githubusercontent.com/2086522/58817987-4e0fa380-85ea-11e9-84cf-89e4ca0b19e8.png)

**Comment by mmenozzi** [comment]
I can confirm this issue.

**Comment by mmenozzi** [comment]
I run this role as follows:
```yaml
- hosts: all
  remote_user: root
  roles:
     - { role: newrelic.newrelic-infra, nrinfragent_config: {license_key: "xxxxxxxxx"} }
```

**Comment by mmenozzi** [comment]
@alejandrodnm I think the regression has been introduced here: https://github.com/newrelic/infrastructure-agent-ansible/commit/25c78c84e8fac4d4bf56c8f140fac7917d4c37cd#diff-7eeda618087b49ae876084ab6c73fdbbR9

I think that defining the `nrinfragent_license_key` variable with `null` value makes the `templates/newrelic-infra.yml.j2` template to compile with an empty `license_key`.

**Comment by alejandrodnm** [comment]
Hi @mmenozzi, thanks for pointing it out, I already created a ticket for this, and we should review it soon.

We'll keep you posted.

**Comment by alejandrodnm** [comment]
Fixed in #70
